packages/mobile/.env.alfajores

```diff
@@ -19,3 +19,4 @@ GETH_USE_FULL_NODE_DISCOVERY=true
 GETH_USE_STATIC_NODES=true
 # Only for development, use with caution
 GETH_START_HTTP_RPC_SERVER=false
+CROWDIN_DISTRIBUTION_HASH=e-f9f6869461793b9d1a353b2v7c
```

Comment:
will we ever need to update this value?

Reply:
i'll check with Paula about this.

i thought having a separate distribution for alfajores/mainnet would be a good idea, even though we only have one source of truth for the translation files we can still choose to publish updates at different times (i.e. alfajores can be a "preview" - maybe this could be useful?)

i also thought it'd be a good idea to bundle this distribution hash with the app itself, in case of future updates to the structure of the translation files we could have the flexibility to serve different distributions for different apps. again, not sure if this is useful for us - what do you think?

Reply:
I removed these from the env files because Paula said she only needs one distribution for all the apps, this value lives in the config file now :)